server/rooms.ts

```diff
@@ -337,7 +337,7 @@ export abstract class BasicRoom {
 		if (this.auth && this.isPrivate === true) {
 			return ' ';
 		}
-		return user.group;
+		return user.group!;
```

Comment:
```suggestion
		return user.group;
```

This seems like the wrong place to assert, to me.